changelogs/unreleased/gh-6043-fiber-wakeup-self.md

```diff
@@ -4,11 +4,11 @@
   NOP on the currently running fiber.
 
   Previously, they allowed to "ignore" the
-  next yield or sleep, leading to unexpected spurious wake-ups. Could lead to a
+  next yield or sleep, leading to unexpected spurious wake-ups. This could lead to a
   crash (in debug build) or undefined behavior (in release build) if called
   right before `fiber.create()` in Lua or `fiber_start()` in C (gh-6043).
```

Comment:
Unusual line breaks. I don't know what a text editor you're using, but I use `gqap` to format a paragraph in Vim.